addons/event/static/src/js/field_event_mail_template_reference.js

```diff
@@ -0,0 +1,28 @@
+/** @odoo-module **/
+
+import field_registry from 'web.field_registry';
+import { FieldReference } from 'web.relational_fields';
+
+
+const EventMailTemplateReferenceField = FieldReference.extend({
+    init: function () {
+        this._super(...arguments);
+        this.noOpen = true;
+    },
+    _getModelIcon: function () {
+        if (this.modelName === 'mail.template') {
+            return '<i class="fa fa-solid fa-envelope"></i>';
+        }
+        return '';
+    },
+    _renderReadonly: function () {
+        this.$el.html(`<span style="display: inline-block; text-align: center; width: 1rem; margin-right: 0.5rem;">
```

Comment:
Why inline html and not a small qweb template ? So that it might be customized / fixed / improved / ...

Reply:
Good idea, had to switch to the older module definitions to access core.qweb in 15.0